nyc_taxis/challenges/indexing-querying.json

```diff
@@ -92,10 +92,22 @@
               "operation-type": "search",
               "body": {
                 "size": 10000,
```

Comment:
I realize this is only the indexing-querying challenge, but still, any reason not to set `size` to 0?

Reply:
I'd rather just drop the challenge, to be honest. especially given all the playing around we're doing in the default challenge.  I'll sneak that one into this change and re-request review, since we won't merge until monday at the earliest anyway